app/Models/Post.php

```diff
@@ -57,6 +59,16 @@ class Post extends Model
      */
     public static $indexes = ['id', 'signup_id', 'campaign_id', 'type', 'action', 'action_id', 'northstar_id', 'status', 'created_at', 'source', 'location'];
 
+    /**
+     * Attributes that can be sorted by.
+     *
+     * This array is manually maintained. It does not necessarily mean that
+     * any of these are actual indexes on the database... but they should be!
+     *
+     * @var array
+     */
+    public static $sortable = ['created_at'];
```

Comment:
Aren't you sorting by `id` as well?

Reply:
This one's a little awkward, because we don't actually choose `orderBy` for this endpoint! But on other endpoints, like campaigns, we'd use this array to decide what `?orderBy=` values are allowed (and then, to decide whether or not we're safe to encode a given columns value into the `cursor` value in a transformer).

Reply:
I bet there's a clearer way of specifying this (especially with the awkward forced query-string on this endpoint), but I haven't stumbled upon it yet. Let me know if anything comes to mind!

Reply:
So we'd include `id` if it was a primary sort option (available to `?orderBy=`), but since it's just our hard-coded secondary sort that we use alongside whatever is chosen as the primary column, it doesn't hit this logic.

Reply:
AH I think for me this would make more sense to me if the comment said something like `Attributes that can be sorted by using the ?orderBy query parameter`

Reply:
Yeah, that's way clearer! ✏️